src/2048-expert/index.html

```diff
@@ -13,6 +13,31 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+            The aim of the game is to get 2048
+            <br>
+            <br>
+            Use the arrow keys to move the tiles
+            <br>
+            Restart a new game by pressing enter
+            <br>
+            Tiles with the same value will get merged after moves
+            <br>
+            <br>When a tile has the value <strong>2048</strong>
```

Comment:
This first line in the dialogue summarises the aim of the game. We don't want to congratulate the user before they have actually won so these 2 lines might not be appropriate pre-game.